sys/include/net/gnrc/udp.h

```diff
@@ -54,6 +54,13 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default UDP port range for dynamic source port allocation
+ */
+#define GNRC_UDP_PORTRANGE_MIN (49152U)
```

Comment:
Since I always need to google what that term means when you mention it I disagree. The doc sufficiently makes it clear IMHO.